easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.2rc2-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,19 @@
+name = 'MVAPICH2'
+version = '2.2rc2'
+
+homepage = 'http://mvapich.cse.ohio-state.edu/overview/mvapich2/'
+description = "This is an MPI 3.0 implementation.  It is based on MPICH2 and MVICH."
+
+toolchain = {'name': 'GCC', 'version': '4.9.3-2.25'}
+
+source_urls = ['http://mvapich.cse.ohio-state.edu/download/mvapich/mv2/']
+sources = [SOURCELOWER_TAR_GZ]
+
+osdependencies = [('libibverbs-dev','libibverbs-devel'),('libibmad-dev','libibmad-devel')]
```

Comment:
@arif-ali @verdurin takes a different approach in #3588, by adding `libibmad` as dependency handled through EasyBuild...

It's not clear to me how closely `libibmad` is tied to what the OS provides; if it is closely tied, this here is the proper approach imho...